internal/web/server.go

```diff
@@ -751,3 +755,17 @@ func isPortConflictError(err error) bool {
 	}
 	return false
 }
+
+func AutoUpdateString(pkg *v1alpha1.Package, disabledStr string) string {
```

Comment:
This function is not web-specific (you are also using it in the commands). Please move it to a new file `internal/clientutils/autoupdate.go`.